docs/sdks/javascript/index.md

```diff
@@ -163,13 +163,13 @@ Communication with the Algorand network is performed using transactions. To crea
 
 ​
 !!! Info
-    Algorand supports many transaction types. To see what types are supported see [Transactions](https://developer.algorand.org/docs/features/transactions/).
+    Algorand supports many transaction types. To see what types are supported see [Transactions](https://developer.algorand.org/docs/get-details/transactions/).
 ​
 # Sign First Transaction
-Before the transaction is considered valid, it must be signed by a private key. Use the following code to sign the transaction.
+Before the transaction is considered valid, it must be signed by a private key. Use the following code to sign the transaction. Once signed, we can extract the transaction ID. You'll use the `txId` to look up the status of the transaction in the following sections.
 ​
```

Comment:
Technically the txid can be pulled from a transaction prior to signing

Reply:
added info